examples/src/main/java/org/apache/ignite/examples/ml/tree/DecisionTreeRegressionTrainerExample.java

```diff
@@ -49,15 +49,15 @@ public static void main(String... args) throws InterruptedException {
                 // Create cache with training data.
                 CacheConfiguration<Integer, Point> trainingSetCfg = new CacheConfiguration<>();
                 trainingSetCfg.setName("TRAINING_SET");
-                trainingSetCfg.setAffinity(new RendezvousAffinityFunction(false, 10));
+                trainingSetCfg.setAffinity(new RendezvousAffinityFunction(false, 1));
```

Comment:
And again, why 1? Doesn't work with multiple partitions?

Reply:
see above